samples/callbacks/sample_dotnetcore_callbacks_[3,).partial.md

```diff
@@ -0,0 +1 @@
+WARNING: The WebSender project is not included in the .NET Core version of the sample
```

Comment:
Would an info be enough?

Reply:
I suppose so, but there's already another info box, so I was thinking about using warning just to have a color distinction. I could switch it, though.